### PR TITLE
feature(0.0.0): implement setLog command

### PR DESCRIPTION
**Pull request by ddivad195**
# feature(0.0.0): implement setLog command

#11: Add a new commnad `$setlog` to allow the logging channel to be set or changed.

image

### CODE REVIEW

File: src/main/kotlin/com/supergrecko/questionbot/commands/ManageCommands.kt [hunk 1]
```diff
@@ -57,6 +57,30 @@ fun manageCommands(config: ConfigService, logService: LogService) = commands {
         }
     }
 
+    command("setLog") {
```
Comment (junlarsen) [inline src/main/kotlin/com/supergrecko/questionbot/commands/ManageCommands.kt]: Could you make the name lowercase for consistency?

File: src/main/kotlin/com/supergrecko/questionbot/commands/ManageCommands.kt [hunk 2]
```diff
@@ -57,6 +57,30 @@ fun manageCommands(config: ConfigService, logService: LogService) = commands {
...
+        description = "Sets the log channel."
+        requiresGuild = true
+        permission = PermissionLevel.ADMIN
+
+        expect(WordArg)
```
Comment (junlarsen) [inline src/main/kotlin/com/supergrecko/questionbot/commands/ManageCommands.kt]: Probably want to use TextChannelArg

File: src/main/kotlin/com/supergrecko/questionbot/commands/ManageCommands.kt (hunk 1 above)
Comment (ddivad195) [reply src/main/kotlin/com/supergrecko/questionbot/commands/ManageCommands.kt]: sure, makes sense

**Comment by junlarsen** [comment]
Closes #11

**Pull request merged**